packs/ac-advanced-maneuvers/Bony_Constriction_AH3kNbdGLYaRjVAr.json

```
@@ -14,9 +14,26 @@
   "system": {
     "description": {
       "gm": "",
-      "value": "<p><strong>Requirements</strong> The constrictor has a creature @Compendium[pf2e.conditionitems.Grabbed]{Grabbed}.</p>\n<hr />\n<p>The skeletal constrictor deals [[/r 12[piercing]]]{12 piercing Damage} to the grabbed creature with its sharp ribs; the creature must attempt a @Check[type:fortitude|dc:resolve(12+@actor.level+@actor.abilities.con.mod)|basic:true] save. If the constrictor is a specialized companion, increase this damage to [[/r 20[piercing]]]{20}. This uses a trained DC using the constrictor's Constitution modifier, or an expert DC (@Check[type:fortitude|dc:resolve(14+@actor.level+@actor.abilities.con.mod)|basic:true]) if the constrictor is specialized.</p>"
+      "value": "<p><strong>Requirements</strong> The constrictor has a creature @UUID[Compendium.pf2e.conditionitems.Item.kWc1fhmv9LBiTuei]{Grabbed}.</p><hr /><p>The skeletal constrictor deals @Damage[12[piercing]] damage to the grabbed creature with its sharp ribs; the creature must attempt a @Check[fortitude|basic|dc:resolve(@actor.flags.pf2e.companionCompendia.fbonyConstrictionDC)] save. If the constrictor is a specialized companion, increase this damage to @Damage[20[piercing]|shortLabel]. This uses a trained DC using the constrictor's Constitution modifier, or an expert DC if the constrictor is specialized.</p>"
```

```suggestion
      "value": "<p><strong>Requirements</strong> The constrictor has a creature @UUID[Compendium.pf2e.conditionitems.Item.kWc1fhmv9LBiTuei]{Grabbed}.</p><hr /><p>The skeletal constrictor deals @Damage[12[piercing]] damage to the grabbed creature with its sharp ribs; the creature must attempt a @Check[fortitude|basic|dc:resolve(@actor.flags.pf2e.companionCompendia.bonyConstrictionDC)] save. If the constrictor is a specialized companion, increase this damage to @Damage[20[piercing]|shortLabel]. This uses a trained DC using the constrictor's Constitution modifier, or an expert DC if the constrictor is specialized.</p>"
```
I still think special statistic for these but we can at least fix that typo.

---

Also note for damage alteration